src/components/common.js

```diff
@@ -310,23 +459,31 @@ const formatGroupLabel = group => (
     }
   }, [group.label]))
 
-const BaseSelect = ({ value, newOptions, id, findValue, maxHeight, ...props }) => {
+const BaseSelect = ({ value, newOptions, id = null, findValue, maxHeight, ...props }) => {
   const newValue = props.isMulti ? _.map(findValue, value) : findValue(value)
+  const [isOpen, setOpen] = useState(false)
+  const menuId = Utils.useUniqueId()
+  const myId = Utils.useUniqueId()
+  const inputId = id || myId
```

Comment:
Can you inline the definition of `myId` here? (`id || Utils.useUniqueId()`) Just curious, because that could avoid us generating the unique id if we already have an id to use, and avoids needing to name the variable.

Reply:
Ha! This is exactly what I tried to do, but then React complained about the number of hooks changing. :)

Reply:
`Utils.useUniqueId()` uses `useRef` under the hood